### PR TITLE
OISP-Frontend and OISP-Websocket-Server configuration independent of VCAP. 

### PR DESCRIPTION
**Pull request by oguzcankirmemis**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -175,12 +175,23 @@ services:
       - websocket-server
       - backend
       - kafka
-    command: ./wait-for-it.sh postgres:5432 -t 300 -- ./wait-for-it.sh redis:6379 -t 300  -- ./wait-for-it.sh kafka:9092 -t 300 -- ./scripts/wait-for-heartbeat.sh backend websocket-server -- ./scripts/docker-start.sh ${RATE_LIMIT}
+    command: ./wait-for-it.sh postgres:5432 -t 300 -- ./wait-for-it.sh redis:6379 -t 300  -- ./wait-for-it.sh kafka:9092 -t 300 -- ./scripts/wait-for-heartbeat.sh backend websocket-server -- ./scripts/docker-start.sh
     volumes:
       - ./data/keys:/app/keys
+      - ./oisp-frontend/public-interface:/app
```
Comment (wagmarcel) [inline docker-compose.yml]: Why do we mount this volume? This is only for debugging, right?

**Comment by oguzcankirmemis** [comment]
You are right, I forgot to remove that line before the actual commit. It is deleted now.

**Comment by oguzcankirmemis** [comment]
Also added the configuration for the websocket-server as another commit

**Comment by arkocal** [comment]
@wagmarcel Are you happy with the changes?

**Pull request merged**